cub/test/catch2_test_device_segmented_sort_keys.cu

```diff
@@ -24,62 +24,20 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  ******************************************************************************/
+
 #include "insert_nested_NVTX_range_guard.h"
 // above header needs to be included first
+#include <cub/device/device_segmented_sort.cuh>
+
 #include "catch2_radix_sort_helper.cuh"
 #include <c2h/catch2_test_helper.h>
 #include <catch2_segmented_sort_helper.cuh>
```

Comment:
Nitpick: we are really inconsistent with our includes.

No change requested

Reply:
Yah, that should really be `"catch2_segmented_sort_helper.cuh"`. I fixed the include.